composer.json

```diff
@@ -11,12 +11,13 @@
     "autoload-dev": {
         "psr-4": {
             "tests\\Libero\\ContentStore\\": "tests/",
-            "tests\\Libero\\JatsContentWorkflow\\": "vendor-extra/JatsContentWorkflow/tests/"
+            "tests\\Libero\\JatsContentWorkflowBundle\\": "vendor-extra/JatsContentWorkflowBundle/tests/"
```

Comment:
so much for unrelated changes? :sweat_smile:

Reply:
https://github.com/libero/content-store/pull/16#discussion_r302076714 😳